pims/cine.py

```diff
@@ -586,6 +730,97 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
+    def save_image_sequence(self, base_dir=None, im_ext='.tif', fmt='06d',
```

Comment:
This is a very useful addition, but I don't think it should go here. Only the `crop` option is specific to the CINE format. Everything else would apply to all 2D sequences in PIMS. The ffmpeg export in particular is already found in the `display` module. (It is not well-documented, but since it has been tested and used more widely, it is better to build on that functionality.)

It would be better to add a new module, maybe called `export`. @danielballan , what do you think?

Could `crop`, and the use of `white_level` and `black_level`, be options for the reader class itself?

Reply:
Ok, let's move it away for now. This will solve the problem of the skimage dependency as well.
Anyway, I had to come back to it, I'm experiencing some trouble with my movies export. I'll work on my own on it and if I arrive to some satisfactory, general result, I'll submit the code, in a separate `export` module.

`crop`, `white_level` and `black_level` are in the `Cine.setup_fields_dict`. They can easily be implemented as options.

Reply:
OK. Be sure to check out the ffmpeg code in `display.py`; it is done in memory so it is relatively fast.

Reply:
The function is removed. I will check `display.py`.